src/models/request/userBulkDeleteRequest.ts

```diff
@@ -0,0 +1,7 @@
+export class UserBulkDeleteRequest {
+    ids: string[];
+
+    constructor(ids: string[]) {
+        this.ids = ids == null ? [] : ids;
+    }
+}
```

Comment:
Related to [this](https://github.com/bitwarden/server/pull/1320#discussion_r631345049) question on the server PR